styles/styles.css

```diff
@@ -0,0 +1,534 @@
+}
+
+.blog-article > p {
+  margin: 3% 0;
+}
+.blog-article > a {
+  text-decoration: none;
+  color: #225fc1;
+}
+
+
+/*Body-Left: Doggo grid*/
+
+.clean-text {
+  text-decoration: none;
+  text-align: left;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.grid {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-gap: 3%;
+  margin-bottom: 10%;
+}
+
+.doggo-grid {
+  border: 1px solid #e8e8e8;
+  display: flex;
+  flex-direction: column;
+}
+.doggo-grid > img {
+  width: 100%;
+  height: 100%;
+}
+.doggo-grid > h3 {
+  color: #1c356d;
+  font-size: 2em;
+  padding: 0 3%;
+}
+.doggo-grid > p {
+  line-height: 1.5;
+  padding: 0 5%;
+}
+.doggo-grid > button {
+  align-self: center;
+  background-color: #81b741;
+  color: #ffffff;
+  padding: 5%;
+  margin: 5% 0;
+  width: 80%;
+  border: 1em;
+  border-radius: .2em;
+  letter-spacing: .1em;
+  font-size: 1em;
+  font-weight: bold;
+}
+
+.more-dog {
+  display: flex;
+  justify-content: center;
+  padding: 5%;
+}
+
+.more-dog > *{
+  text-decoration: none;
+  color: #1c356d;
+  background-color: #e8e8e8;
+  font-weight: bold;
+  padding: 1% 4%;
+  border-radius: .3em;
+}
+
+
+/*Footer*/
+
+.footer{
+  background-color: #133e70;
+  text-align: center;
+  color: #ffffff;
+  padding: 5%;
+  margin-top: 10%;
+}
+
+
+.footer > p {
+  padding: 2% 30%;
+  text-align: center;
+  line-height: 1.5;
+}
+
+.footer-navigation > a {
+  text-decoration: none;
+  color: #ffffff;
+  font-weight: bold;
+  padding: 0 1%;
+}
+
+.footer-navigation > a:first-child {
+  padding: 0 1% 0 0;
+}
+
+
+#logo-footer {
+  width: 30%;
+}
+
+/* Dog Page Stylings*/
+
+#doggo {
+  padding: 5% 12%;
+  color: #1c356d;
+}
+
+.dog-list {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: 5%;
+  padding: 0 12% 10% 12%;
+}
+
+.dog-item {
+  border: 1px solid #e8e8e8;
+  display: flex;
+  flex-direction: column;
+}
+.dog-item > img {
+  width: 100%;
+}
+.dog-item > img:hover {
+  cursor: pointer;
+}
+.dog-item > h3 {
+  color: #1c356d;
+  font-size: 2em;
+  padding: 0 3%;
+}
+.dog-item > p {
+  line-height: 1.5;
+  padding: 5%;
+}
+.dog-item > button {
+  align-self: center;
+  background-color: #418c27;
+  color: #ffffff;
+  padding: 5%;
+  margin: 5% 0;
+  width: 70%;
+  border-radius: .3em;
+  letter-spacing: .1em;
+  font-size: 1em;
+  font-weight: bold;
+}
+.dog-item > button:hover {
+  cursor: pointer;
+}
+.dog-item > button:focus {
+  outline: none;
+}
+
+
+/*Blog Stylings*/
+
+.blog-body {
+  padding: 1.5% 12% 0% 12%;
+  margin: auto;
+}
+
+.article {
+  display: grid;
+  grid-template-columns: repeat(2, 25%);
+  grid-gap: 3%;
+  margin-bottom: 10%;
+  align-content: center;
+  color: #1c356d;
+  }
+
+.article > img {
+  width: 100%;
+  height: 100%;
+}
+
+.context {
+  margin: 3% 0;
+}
+
+/* Styling for Checkout page */
+.checkout {
+  color: #133e70;
+  padding: 2% 12%;
+  align-items: flex-start;
+  align-content: flex-start;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
```

Comment:
This is affecting your css for the checkout button. grid columns seem unnecessary.